client/src/App.js

```diff
@@ -4,24 +4,22 @@ import About from "./pages/About";
 import Donations from "./Donations";
 import ThankYou from "./pages/ThankYou";
 import Home from "./pages/Home";
-import About from "./pages/About";
```

Comment:
Checking should this be removed if we are using about component in our react routes?

Reply:
Hi @Monika39-maker, I don't think this is resolved. Please could you take a look